engine/lib/users.php

```diff
@@ -449,10 +449,16 @@ function user_avatar_hook($hook, $entity_type, $returnvalue, $params) {
 	$size = $params['size'];
 
 	if (isset($user->icontime)) {
-		return "avatar/view/$user->username/$size/$user->icontime";
```

Comment:
We can leave the URL alone and forward from the resource view. Not sure how browsers will respond to caching with 301 headers...